lib/aruba/cucumber/hooks.rb

```diff
@@ -5,7 +5,7 @@
 
 if Aruba::VERSION >= '1.0.0'
   Around do |_, block|
-    Aruba.platform.with_environment(&block)
+    with_environment(&block)
```

Comment:
The `Aruba.platform.with_environment` version clears the entire environment, including things like `PATH`, causing a lot of cukes to fail because they can't find executables, among other things.